internal/gitserver/client.go

```diff
@@ -494,9 +494,10 @@ type Client interface {
 	// LogReverseEach runs git log in reverse order and calls the given callback for each entry.
 	LogReverseEach(ctx context.Context, repo string, commit string, n int, onLogEntry func(entry gitdomain.LogEntry) error) error
 
-	// RevList makes a git rev-list call and iterates through the resulting commits, calling the provided
-	// onCommit function for each.
-	RevList(ctx context.Context, repo string, commit string, onCommit func(commit string) (bool, error)) error
+	// RevList makes a git rev-list call and returns up to count commits. if nextCursor
+	// is non-empty, it is used as the starting point for the next call, feed it to commit
```

Comment:
I think there is a grammtical issue (feed it to commit). Maybe just "to iterate" like the next line?

```suggestion
	// is non-empty, it is used as the starting point for the next call, use it to iterate
```